core/src/components/picker-internal/picker-internal.tsx

```diff
@@ -550,6 +551,21 @@ export class PickerInternal implements ComponentInterface {
           tabindex={-1}
           inputmode="numeric"
           type="number"
+          onKeyDown={(ev: KeyboardEvent) => {
+            /**
+             * The "Enter" key represents
+             * the user submitting their time
+             * selection, so we should blur the
+             * input (and therefore close the keyboard)
```

Comment:
I don't think we can safely make that assumption because the user may still want to change the AM/PM values after confirming the numeric time selection.

While the exact UI on Android isn't the same, the keyboard dismiss behavior is consistent with the proposed changes:


https://github.com/ionic-team/ionic-framework/assets/2721089/ffdd7e4f-99d3-4a10-8e78-b855af502b04

Reply:
Fair enough!